src/access_nri_intake/source/builders.py

```diff
@@ -543,8 +544,76 @@ def parser(cls, file):
             return ncinfo_dict
 
         except Exception:
+            if "NotImplementedError" in traceback.format_exc():
+                raise NotImplementedError(f"Ready to implement for {file}")
             return {INVALID_ASSET: file, TRACEBACK: traceback.format_exc()}
 
+    @classmethod
+    def parse_access_ncfile(
+        cls, file: str, time_dim: str = "time"
+    ) -> _AccessNCFileInfo:
```

Comment:
This looks pretty similar to the standard `parse_access_ncfile` to me, just with a change to the TimeParser object being used. Can the time parser be made an optional kwarg of the standard `parse_access_file` and we toss this version, or is there something else I've missed?

Reply:
Bear with me - I'm gonna rename a few slightly misnamed functions & hopefully the intention should become clearer